internal/controllers/machinedeployment/machinedeployment_sync_test.go

```diff
@@ -550,9 +550,8 @@ func TestComputeDesiredMachineSet(t *testing.T) {
 			Annotations: map[string]string{"top-level-annotation": "top-level-annotation-value"},
 		},
 		Spec: clusterv1.MachineDeploymentSpec{
```

Comment:
Can we have test coverage please that MinReadySeconds gets propagated? Currently this PR just drops all MinReadySeconds fields in these tests